packages/react-components/src/visually-hidden/src/VisuallyHidden.tsx

```diff
@@ -1,23 +1,30 @@
 import "./VisuallyHidden.css";
 
 import { Box } from "../../box";
-import { elementType, oneOfType, string } from "prop-types";
-import { forwardRef } from "react";
-import { mergeProps } from "../../shared";
+import { ElementType, ForwardedRef, ReactElement, ReactNode } from "react";
+import { forwardRef, mergeProps } from "../../shared";
 
-const propTypes = {
+export interface InnerVisuallyHiddenProps {
     /**
      * An HTML element type or a custom React element type to render as.
      */
-    as: oneOfType([string, elementType])
-};
+    as: ElementType;
+    /**
```

Comment:
Wasn't specified before, required or optional in this case?

Reply:
Optionnal